lemur/domains/service.py

```diff
@@ -65,6 +76,10 @@ def create(name, sensitive):
     :param sensitive:
     :return:
     """
+    domain = get_by_name(name)
```

Comment:
shall we revise the comments to clarify the revised behavior of `create`?

Reply:
Good call